Modelica/Mechanics/MultiBody/Examples/Systems/RobotR3/Utilities/Motor.mo

```diff
@@ -3,11 +3,12 @@ model Motor "Motor model including current controller of r3 motors"
   extends Electrical.Machines.Icons.Machine;
   parameter SI.Inertia J(min=0)=0.0013 "Moment of inertia of motor";
   parameter Real k=1.1616 "Gain of motor";
-  parameter Real w=4590 "Time constant of motor";
-  parameter Real D=0.6 "Damping constant of motor";
+  parameter Real w(unit="s-1")=4590 "Time constant of motor";
+  parameter Real D(unit="1")=0.6 "Damping constant of motor";
   parameter SI.AngularVelocity w_max=315 "Maximum speed of motor";
   parameter SI.Current i_max=9 "Maximum current of motor";
```

Comment:
```suggestion
```

Remove if not used.

Reply:
I'm not confident in removing that in this PR.

Plotting the different axis*.motor.emf.i and axis*.motor.speed.w; indicates that the speed is within the bounds - but that the current peaks above the maximum. (For axis3 it is close to 15 A, so significantly above.)
I don't know - it could even be different for the different axis (it's worse for axis 1, 2, 3 and they are different.)

Reply:
I hope @MartinOtter can give feedback.